### ISSUE TITLE
Exported from roboflow.ai with 3 categories!

### ISSUE THREAD

**Issue by NVFALCON** [zylo117/Yet-Another-EfficientDet-Pytorch#364]
Im Exported from roboflow.ai with 2 class, Did I configure it properly?

![Screenshot from 2020-06-02 15-57-32](https://user-images.githubusercontent.com/35072229/83501117-e08e9f80-a4e9-11ea-861d-e5dae66ff4dd.png)

![Screenshot from 2020-06-02 15-59-28](https://user-images.githubusercontent.com/35072229/83501314-22b7e100-a4ea-11ea-89ce-fb829d6db4bb.png)

**Comment by NVFALCON** [comment]
![Screenshot from 2020-06-02 16-06-54](https://user-images.githubusercontent.com/35072229/83502022-331c8b80-a4eb-11ea-8324-384b7ca11fcc.png)

**Comment by zylo117** [comment]
what is pr-tb? is that a valid class?
If so, it should be category id 1 and obj_lists should contains 3 objects.

**Comment by NVFALCON** [comment]
Yes pr = "paragraph" tb ="Table" i want model learn detect it in document!
In dataset I had only pr and tb, but when push it to roboflow.ai they want have value "Annotation Class" im push it = "pr-tb" so I see it in json file i don't now how to config for yml is correct! 
![Screenshot from 2020-06-03 09-01-50](https://user-images.githubusercontent.com/35072229/83587634-b5539100-a579-11ea-9f8d-7e3a38f62bed.png)
![Screenshot from 2020-06-03 09-02-10](https://user-images.githubusercontent.com/35072229/83587639-b7b5eb00-a579-11ea-8874-73d84267649b.png)

**Comment by NVFALCON** [comment]
I try train with " python train.py -c 0 --batch_size 12 --optim sgd --lr 8e-2" loss is very hight and result is:
![img_inferred_d0_this_repo_0](https://user-images.githubusercontent.com/35072229/83587959-7d991900-a57a-11ea-97a2-7207038a0922.jpg)

When i train "Train a custom dataset with pretrained weights " d7  model is stop in epochs 1 not run val???

**Comment by zylo117** [comment]
you should run the tutorial first

**Comment by NVFALCON** [comment]
ok thanks! :)
